agent/mapcall/src/main/java/randoop/instrument/Premain.java

```diff
@@ -44,8 +44,10 @@
    */
   public static void premain(String agentArgs, Instrumentation inst) throws IOException {
 
-    System.out.format(
-        "In premain, agentargs ='%s', " + "Instrumentation = '%s'%n", agentArgs, inst);
+    if (verbose) {
```

Comment:
This seems unrelated to the rest of the changes.  How about making this change on master instead?  Then it won't show up in the diffs.